### PR TITLE
Add known dwl status bar options to README

### PR DESCRIPTION
**Pull request by fauxmight**

### CODE REVIEW

**Comment by JoshVanL** [comment]
Hi, is adding [somestatus](https://github.com/joshvanl/somestatus) useful to include?

**Comment by fauxmight** [comment]
@JoshVanL While I lack no respect for your work, it would appear that [somestatus](https://github.com/JoshVanL/somestatus) contributes non-dwl-specific data to a dwl-specific status bar. It probably belongs in another category.

I personally can weasel out of making a final decision on that: I have no commit rights to this repo. : )

**Comment by JoshVanL** [comment]
@fauxmight no worries, makes sense :+1:

**Comment by Sevz17** [comment]
> Hi, is adding [somestatus](https://github.com/joshvanl/somestatus) useful to include?

I think you should contact to somebar's maintainer to include in its readme

File: README.md [hunk 1]
```diff
@@ -70,6 +70,14 @@ Information about selected layouts, current window title, and selected/occupied/
 If your startup command is a shell script, you can achieve the same inside the script with the line
 
     exec <&-
+	
+Existing dwl-specific status bars and dwl-specific scripts for other status bars include:
+- [somebar](https://sr.ht/~raphi/somebar/) status bar designed for dwl
+- [dtaobarv2.sh](https://cdn.discordapp.com/attachments/792078050024095745/862428883423723560/dtaobarv2.sh) for use with [dtao](https://github.com/Moulberry/dtao) (See "Pinned Messages" on the "customizations" channel of the [dwl Discord server](https://discord.gg/jJxZnrGPWN) for details.)
```
Comment (Sevz17) [inline README.md:76]:
```suggestion
- [dtaobarv2.sh](https://cdn.discordapp.com/attachments/792078050024095745/862428883423723560/dtaobarv2.sh) for use with [dtao](https://github.com/djpohly/dtao) (See "Pinned Messages" on the "customizations" channel of the [dwl Discord server](https://discord.gg/jJxZnrGPWN) for details.)
```

**Comment by fauxmight** [comment]
@Sevz17 Thanks for the tip on 'force push.' Fixed added white space as well.

**Comment by Sevz17** [comment]
Thanks!

**Pull request merged**